### PR TITLE
Minor English Fixes

### PR DESCRIPTION
**Pull request by maieul**

### CODE REVIEW

File: reledmac.dtx [hunk 1]
```diff
@@ -428,10 +428,10 @@
 % \changes{v2.15.1}{2016/09/07}{Fix \protect\cs{edindex} in tabular environments.}
 % \changes{v2.15.2}{2016/09/12}{Fix bug with \protect\package{fancyhdr} package 3.8 and later.}
 % \changes{v2.15.3}{2016/09/26}{Fix bug with \protect\cs{section} in optional argument of \protect\cs{pstart} and empty line before \protect\cs{pend} (bug added in v2.8.2).}
-% \changes{v2.15.3}{2016/09/26}{Simplification of the code concerning sectioning command.}
-% \changes{v2.16.0}{2016/10/01}{Display warning message if using a too older versions of \protect\LaTeX.}
+% \changes{v2.15.3}{2016/09/26}{Simplification of the sectioning command code.}
+% \changes{v2.16.0}{2016/10/01}{Display warning message if using a version of \protect\LaTeX that is too old.}
 % \changes{v2.16.0}{2016/10/01}{Deleting dead code.}
-% \changes{v2.16.0}{2016/10/01}{New options of \protect\cs{fnpos} and \protect\cs{mpfnpos} to set a customized alternance between familiar and critical footnotes.}
+% \changes{v2.16.0}{2016/10/01}{New options of \protect\cs{fnpos} and \protect\cs{mpfnpos} to set a customized alternative between familiar and critical footnotes.}
```
Comment (maieul) [inline reledmac.dtx]: no, it is nore alternative, nore alternance, but alternation

File: reledmac.dtx [hunk 2]
```diff
@@ -3937,8 +3937,8 @@
 \RequirePackage{etoolbox}
 \@ifl@t@r\fmtversion{2015/10/01}
  {\ifboolexpr{not test{\@ifl@t@r\fmtversion{2016/03/31}} or (test{\ifdefstring{\fmtversion}{2016/03/31}} and test {\ifnumless{\patch@level}{3}})}%
-   {\PackageWarning{reledmac}{You use a LaTeX version older than 2016/03/31 patch 3.%
-   \MessageBreak You are strongly encouraged to use a newer version}}%
+   {\PackageWarning{reledmac}{You using a LaTeX version older than 2016/03/31 patch 3.%
```
Comment (maieul) [inline reledmac.dtx]: are your sur of "you using" ?

File: reledmac.dtx [hunk 3]
```diff
@@ -13744,9 +13744,9 @@
 %    \end{macrocode}
 % \end{macro}
 % \begin{macro}{\do@feet@custom@order}
-% \cs{do@feet@custom@order} is called when \cs{@fnpos} is nore `familiar-critical' nore `critical-familiar', that is when the order is more complex.
-% In this case, people must define the order for all footnotes series.
-% If they don't, \LaTeX\ could make infinite run.
+% \cs{do@feet@custom@order} is called when \cs{@fnpos} is neither `familiar-critical', nor `critical-familiar', that is, when the order is more complex.
+% In this case, people must define the order for all footnote series.
+% If they don't, \LaTeX\ could perform infinite runs.
```
Comment (maieul) [inline reledmac.dtx]: no, it is only one run, which is infinite, and not a infinite number of runs

File: reledmac.dtx [hunk 4]
```diff
@@ -13764,8 +13764,8 @@
 %    \end{macrocode}
 % \end{macro}
 % \begin{macro}{\do@Xfeet}
-% \protect\cs{do@Xfeet} is the code extending \protect\cs{@makecol} to cater for
-% the extra critical feet. 
+% \protect\cs{do@Xfeet} is the code extending \protect\cs{@makecol} to cater to
+% the extra critical feet % (do you mean "footers" or "footnotes"?).
```
Comment (maieul) [inline reledmac.dtx]: yes, it is footnotes (but this txt is not from me, but from original author)

File: reledmac.dtx [hunk 5]
```diff
@@ -13913,7 +13913,7 @@
 % \end{macro}
 % \end{macro}
 %
-% \subsection{Patching standard output's commands}
+% \subsection{Patching output's standard commands}
```
Comment (maieul) [inline reledmac.dtx]: no. It is the command for the standard output, not the standard commands for the output

File: reledmac.dtx (hunk 1 above)
Comment (bishopcranmer) [inline reledmac.dtx]: Then the word is "alteration"

File: reledmac.dtx (hunk 2 above)
Comment (bishopcranmer) [inline reledmac.dtx]: Sorry - "You are using a ..."

File: reledmac.dtx (hunk 3 above)
Comment (bishopcranmer) [inline reledmac.dtx]: Ah, I see, then "\LaTeX\ could perform an infinite run."

File: reledmac.dtx (hunk 4 above)
Comment (bishopcranmer) [inline reledmac.dtx]: Then it should be "footnotes". But if it's from the original author, you may wish to keep it like that for nostalgic reasons.

File: reledmac.dtx (hunk 5 above)
Comment (bishopcranmer) [inline reledmac.dtx]: I misunderstood.

File: reledmac.dtx (hunk 1 above)
Comment (maieul) [inline reledmac.dtx]: no, "alteration" is (bad) modification. My problem is "do we have all critical footnotes, then familiar footnotes, or do we have a series of critical footnotes, then a series of familiar footnotes, then a series of critical footnotes, then a series of familiar footnotes… or an other pattern"

**Comment by maieul** [comment]
thansk, I have asked a question on "alteration"

File: reledmac.dtx (hunk 1 above)
Comment (maieul) [inline reledmac.dtx]: " to set a customized order for familiar and critical footnotes"… I think that will be enough ;)

**Comment by bishopcranmer** [comment]
I think that the word you may want is “variant”. Google translate gives me the French, which is (interestingly enough) “variante”. The standard definition for “variant” is "a form or version of something that differs in some respect from other forms of the same thing or from a standard”.

Another possibility that is quite subtle is the word “variation”, which is “variation” in French (per Google). Its definition is "a change or difference in condition, amount, or level, typically with certain limits”.

My “gut” says that “variant” is probably best—assuming I understand your purpose!  :-)

M

File: reledmac.dtx (hunk 1 above)
Comment (bishopcranmer) [inline reledmac.dtx]: Wait - that's too simple a solution!!!    :-)

For the record, I think that the word you may want is “variant”. Google translate gives me the French, which is (interestingly enough) “variante”. The standard definition for “variant” is "a form or version of something that differs in some respect from other forms of the same thing or from a standard”.

Another possibility that is quite subtle is the word “variation”, which is “variation” in French (per Google). Its definition is "a change or difference in condition, amount, or level, typically with certain limits”.

My “gut” says that “variant” is probably best—assuming I understand your purpose!  :-)

M

**Comment by maieul** [comment]
no, the problem is NOT a problem of variante, but a problem of order.

**Comment by bishopcranmer** [comment]
Actually, your simple solution is best . . .

M

**Comment by maieul** [comment]
many thanks for all your correction.
maybe, if it is too complex for you to do the commit / pull request, you could simply do as Rebecca does, thats is sending me an email with the correction to do ?

**Comment by bishopcranmer** [comment]
I am too stubborn to quit!  :-)

**Pull request merged**